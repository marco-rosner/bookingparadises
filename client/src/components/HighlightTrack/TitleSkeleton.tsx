import React from "react";
import { Skeleton } from "@mui/material";

export const TitleSkeleton = (): React.ReactElement => (
    <Skeleton width="30%" sx={{ marginLeft: '30px', fontSize: '2rem'}} />
)