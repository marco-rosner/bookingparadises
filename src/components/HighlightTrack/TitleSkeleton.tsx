import { Skeleton } from "@mui/material";

export const TitleSkeleton = () => (
    <Skeleton width="30%" sx={{ marginLeft: '30px', fontSize: '2rem'}} />
)