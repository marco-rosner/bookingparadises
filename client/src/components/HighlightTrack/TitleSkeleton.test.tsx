import renderer from 'react-test-renderer'
import { TitleSkeleton } from './TitleSkeleton';

describe("TitleSkeleton", () => {
    test("snapshot", () => {
        const snapshot = renderer
            .create(<TitleSkeleton />)
            .toJSON();

        expect(snapshot).toMatchSnapshot();
    })
})