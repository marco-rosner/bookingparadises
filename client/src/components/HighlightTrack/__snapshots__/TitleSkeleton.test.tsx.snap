// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleSkeleton snapshot 1`] = `
<span
  className="MuiSkeleton-root-eEyfya csqclO MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse"
  style={
    {
      "height": undefined,
      "width": "30%",
    }
  }
/>
`;
